templates/admin/tabs.php

```diff
@@ -4,15 +4,15 @@
 	'dashboard'   => [ 'url' => menu_page_url( $_GET['page'], false ), 'text' => __( 'Dashboard', 'anycomment' ) ],
 	'social'      => [
 		'url'  => menu_page_url( $_GET['page'], false ) . '&tab=social',
-		'text' => __( 'Social', 'anycomment' )
+		'text' => __( 'Social Login', 'anycomment' )
```

Comment:
Hm, just think.. Can we use "Socials"?

Reply:
I'm trying to find a word that makes sense in english. How about 'Authentication'?

Reply:
Hm, this is too abstract, but it seems to be better as we have WordPress options as well and this is not social related anymore. So let's agree on Authentication :+1:

Reply:
```suggestion
		'text' => __( 'Authentication', 'anycomment' )
```